### DISCUSSION TITLE
Periodic multiscale tesselations

### DISCUSSION THREAD

**Discussion by lreigbua** [neperfepx/neper#446] [Q&A] [answered]
I am using this command on Neper version 4.4.1:

` neper -T -n 3::2::from_morpho -periodicity all -id 1::1 -morpho "gg::gg::lamellar(w=0.1:0.05)" -o Polycrystal_laminated_3scale`

It gives the error:

`Error  : Option `-periodic' is not available for multiscale tessellations.`

Is this still not supported or am I missing something?

**Comment by rquey** [comment, marked as answer]
Not supported yet.

**Reply by IdaRlt** [reply]
Hello Romain,
I would be interested too in such a feature. Is there a chance it might appear in the next versions of Neper ?

**Reply by rquey** [reply]
The chances are slim...

It's not something I need, or can prioritize (unfortunately).

**Reply by IdaRlt** [reply]
Ah, too bad for me.  Thank you for your quick answer.
